10/umbraco-workflow/getting-started/approval-thresholds.md

```diff
@@ -0,0 +1,46 @@
+# Approval thresholds
+
+{% hint style="info" %}
+This feature requires a license - learn more about [Workflow's licensing model](https://umbraco.com/products/umbraco-workflow)
```

Comment:
We have an article about the licensing in the docs as well: https://docs.umbraco.com/umbraco-workflow/licensing 
Does it make sense to link to that instead perhaps?

Reply:
Let's keep the link to .com.